.github/workflows/lint-test.yaml

```diff
@@ -0,0 +1,29 @@
+name: Lint
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.9'
+
+      - name: Install bash
+        run: sudo apt-get update && sudo apt-get install -y bash
```

Comment:
bash should be installed on this runner.
We didn't install it here: https://github.com/kubeflow/trainer/blob/master/.github/workflows/test-python.yaml
However, we still use it in the Makefile: https://github.com/kubeflow/trainer/blob/master/Makefile#L11